solana-mainnet-metadata/frax.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Frax",
+    "symbol": "FRAX",
+    "description": "Frax Token",
+    "image": "https://static.frax.com/images/tokens/frax.png",
+    "extensions": {
+      "website": "https://frax.finance/",
```

Comment:
are we still using frax.finance as the url?

Reply:
I'll update the url , also queue tx on squad to update metadata